include/SimpleTextFloat.h

```diff
@@ -47,7 +47,10 @@ class LMMS_EXPORT SimpleTextFloat : public QWidget
 
 	void showWithDelay(int msecBeforeDisplay, int msecDisplayTime);
 
-	void setVisibilityTimeOut(int msecs);
+	inline void showWithTimeout(int msec)
```

Comment:
```suggestion
	void showWithTimeout(int msec)
```
Member functions with in-class definitions are already implicitly inline.
https://isocpp.org/files/papers/N4860.pdf#subsection.11.4.1